demo/rv/README.md

```diff
@@ -75,8 +90,8 @@ Refer the [Docker Commands](../README.md/#docker-commands) to start the service.
 # Allowlist and Denylist Configuration
 
 - RV provides the option to allow and deny requests based on the owner, manufacturer and reseller public keys and based on the GUID used in the device ownership voucher
-header. 
-- To add entries to these allowlist and denylist, update the `config.properties` file in `<fido-iot-src>/storage/storage-rv-sample/src/main/resources` location.
+header.
+- To add entries to these allowlist and denylist, update the `config.properties` file in `<fido-iot-src>/storage/storage-samples/storage-rv-sample/src/main/resources` location and rebuild the code and restart the docker or update the `config.properties` in `demo/rv` and restart the docker.
```

Comment:
**'in `demo/rv`'** is not required, as it is the current directory in which this README file is present.

Reply:
It might not be obvious that 'demo/rv' is the location, if its removed, since the statement talks about editing 'config.properties' at 2 different locations.